packages/web/src/components/play-bar/desktop/PlayBar.js

```diff
@@ -393,6 +396,9 @@ class PlayBar extends Component {
                 isPlaying={isPlaying && !isBuffering}
                 isDisabled={!uid && !collectible}
                 includeTimestamps
+                playbackRate={
+                  isLongFormContent ? playbackRateValueMap[playbackRate] : 1
```

Comment:
Just wondering if we should always have playback rate set correctly in state so we don't have to make checks on what type of content we are dealing with?

Reply:
Additionally do we need to use the value map all over the place or make that part of a selector?

Reply:
yea it gets a bit annoying so maybe it could be a separate selector, but i like having a selector for the string for rendering out svgs and other UI things

Reply:
Can possibly have two selectors or would that be just as confusing?